src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+public enum Colors {
+    WHITE {
+        @Override
+        public String toString() {
+            return "White";
+        }
+    },
```

Comment:
```suggestion
    WHITE,
```
fix for each color, it's redundant